projects/SelfTest/Baselines/junit.sw.approved.txt

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<testsuitesloose text artifact
->
+<testsuites>
+  <properties>
+    <property name="random-seed" value="1"/>
+  </properties>
+loose text artifact
```

Comment:
I don't understand if there is a "hard" expectation on what should happen with `loose text artifact` here

Reply:
I guess the only expectation is that it will be there :)